requirements/test.txt

```diff
@@ -5,7 +5,7 @@ cryptography==3.3.1; platform_machine!="i686" and python_version<"3.9" # no 32-b
 freezegun==1.1.0
 mypy==0.910; implementation_name=="cpython"
 mypy-extensions==0.4.3; implementation_name=="cpython"
-proxy.py==2.3.1
+proxy.py==3.4.0
```

Comment:
Looks like this was supposed to be v2.4.0 but it's already `proxy.py ~= 2.4.4rc3`.